docs/source/conf.py

```diff
@@ -82,3 +82,11 @@
 # rst_prolog = """
 # .. include:: special.rst
 # """
+
+rst_prolog = """
```

Comment:
There is a subtle bug if we don't declare this where the build will only work if you clean. Anyway, it's worth it because we use this for a lot of files, so we might as well generate for all